shopinvader_product_stock_state/models/shopinvader_variant.py

```diff
@@ -14,7 +14,7 @@ def _prepare_stock_data(self):
         if "state" in config:
             res["state"] = self.stock_state
         if config == "only_state" or (
-            config == "state_and_low_qty" and res["state"] != "in_limited_stock"
+            config == "state_and_low_qty" and res["state"] == "in_stock"
```

Comment:
Small question in case that the product is out of stock, maybe we should force to zero the qty?
Indeed if in odoo we have negative quant, it will show a negative qty on the site (before the field was not here)
It will be interesting to have the feedback of @thibaultrey